src/Ilios/CoreBundle/Entity/User.php

```diff
@@ -1325,23 +1325,52 @@ public function getEncoderName()
     }
     
     /**
-     * Get all the schools an user is affiliated with
-     * This is used in granting view permissions for faculty
-     * in schools that are a users secondary cohort
+     * Get all the schools an user is affiliated with so we can match
+     * permissions.
      *
      * @return ArrayCollection[School]
      */
     public function getAllSchools()
     {
-        $undeletedCohorts = $this->getCohorts()->filter(function (CohortInterface $cohort) {
+        $cohortSchools = $this->getCohorts()->filter(function (CohortInterface $cohort) {
             return !$cohort->isDeleted();
-        });
-        $schools = $undeletedCohorts->map(function (CohortInterface $cohort) {
+        })->map(function (CohortInterface $cohort) {
             return $cohort->getProgramYear()->getProgram()->getSchool();
```

Comment:
Ah hell.  Good catch, I got a little javascripty there for sure.  Where this also probably would have failed.